### ISSUE TITLE
When "play automatically" is off (advanced options), show video thumbnail

### ISSUE THREAD

**Issue by matt873** [libre-tube/LibreTube#5805]
### Describe your suggested feature

Video thumbnail should be shown instead of initial part of the video. This looks more appropriate behaviour and as a way to view thumbnail before starting a video manually.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
There's unfortunately no easy way to accomplish this, so it's out of scope.
